manager/plugin/db.go

```diff
@@ -107,51 +175,83 @@ func (p *pluginDB) DeletePlugin(name string) error {
 	opts := &sql.TxOptions{Isolation: sql.LevelDefault}
 	tx, err := p.conn.BeginTx(p.ctx, opts)
 	if err != nil {
-		log.Info().Str("module", "db").Err(err)
+		log.Error().Str("module", "db").Msgf("DeletePlugin > conn.BeginTx Error: %s", err)
 		return err
 	}
 
 	_, err = tx.Exec("DELETE FROM plugin WHERE name = ?", name)
 	if err != nil {
-		log.Info().Str("module", "db").Err(err)
+		log.Error().Str("module", "db").Msgf("DeletePlugin > tx.Exec Error: %s", err)
 		return err
 	}
 
 	if err = tx.Commit(); err != nil {
-		log.Info().Str("module", "db").Err(err)
+		log.Error().Str("module", "db").Msgf("DeletePlugin > tx.Commit Error: %s", err)
 		return err
 	}
 
 	return nil
 }
 
-func (p *pluginDB) UpdatePlugin() error {
-	// TODO:
+func (p *pluginDB) UpdatePlugin(name string, isEnabled bool) error {
```

Comment:
`UpdatePlugin` is too general, so I think it will be good changing the name on next time.

Reply:
I asked chatgpt, and after a lot of back and forth, UpdateInstalledPlugin() was chosen as a replacement for UpdatePlugin(). If you need to change UpdatePlugin(), how about UpdateInstalledPlugin()?

Reply:
I just thought that it could be better to have more precise name because this function just touches `isEnabled` field. So, I thought like `ChangePluginEnable`. 🤔 

But if you have some plan to add more fields in this function, `UpdatePlugin` seems ok 👍

Reply:
> I just thought that it could be better to have more precise name because this function just touches `isEnabled` field. So, I thought like `ChangePluginEnable`. 🤔
> 
> But if you have some plan to add more fields in this function, `UpdatePlugin` seems ok 👍

@rootwarp 
I explicitly rename it. I guess that's better for now.